target/linux/ipq40xx/base-files/etc/board.d/02_network

```diff
@@ -55,6 +55,11 @@ ipq40xx_setup_interfaces()
 		ucidef_add_switch "switch0" \
 			"0u@eth0" "3:lan" "4:lan"
 		;;
+	mobipromo,cm520-79f)
+		ucidef_set_interfaces_lan_wan "eth0" "eth1"
+		ucidef_add_switch "switch0" \
+			"0u@eth0" "3:lan:2" "4:lan:1"
```

Comment:
As far as I know, ucidef_add_switch will already set up "lan" here, it should be enough to set "wan" explicitly before:
ucidef_set_interfaces_wan "eth1"

Reply:
Done